### DISCUSSION TITLE
Zooming and panning with a mouse

### DISCUSSION THREAD

**Discussion by Carnoisseur1979** [nolimits4web/swiper#6554] [Ideas]
While the zoom module using touch on mobile is great, when using a mouse the current 'click and drag' behaviour isn't what some users expect. We'd like it if once the user has triggered zooming in (by double clicking or another event triggering the `swiper.zoom.in` method), the user can pan across the zoomed image without needing to hold down the mouse button.

See [the demos on this page](http://www.jacklmoore.com/zoom/). Panning on the image on the left shows what I mean, except that triggers zoom on hover.

Related to that, it would also be nice to configure how the zoom is triggered instead of double click like currently done. That example shows zooming on hover or single click (note - single click down then drag, not needing to lift the mouse button first). See #5605.

I mainly wanted to ask if something like this is possible currently? If not, how do you think it should be implemented? I'd be happy to try and send a PR.

**Comment by webdiva69** [comment]
Would love this as well!
